e2e_tests/tests/experiment/experiment.py

```diff
@@ -952,7 +956,7 @@ def verify_completed_experiment_metadata(
         # When the experiment completes, all slots should now be free. This requires terminating the
         # experiment's last container, which might take some time (especially on Slurm where our
         # polling is longer).
-        max_secs_to_free_slots = 300 if api_utils.is_hpc() else 30
+        max_secs_to_free_slots = 300 if api_utils.is_hpc(sess) else 30
```

Comment:
is this the only place you have caching issues?  Or did I miss another place while skimming through the changes?

Reply:
Resolved OOB, this plus one other place I also missed and am adding.